renderer/src/pages/wallet/ConfirmTransfer.tsx

```diff
@@ -0,0 +1,45 @@
+import BorderedBox from 'src/components/BorderedBox'
+import Button from 'src/components/Button'
+import Text from 'src/components/Text'
+import { Wallet } from 'src/hooks/StationWallet'
+import { formatFilValue } from 'src/lib/utils'
+
+const ConfirmTransfer = ({
+  walletBalance,
+  hide,
+  transferAllFundsToDestinationWallet
+}: {
+    walletBalance: Wallet['walletBalance'];
```

Comment:
Could we also write `Wallet.walletBalance`?

Reply:
Sadly, no 
https://www.totaltypescript.com/why-you-cant-use-dot-notation-on-types